### PR TITLE
Add README_AR.md

### PR DESCRIPTION
**Pull request by Majalian**
Readme arabic translation

### CODE REVIEW

**Comment by BaseMax** [comment]
Mujahid, Thank you so much.

**Comment by BaseMax** [comment]
A mistake my friend, you modified main README.md, But we need a new file. 
for e.g: README_AR.md

**Comment by jbampton** [comment]
Can you give me admin on your fork ?   I can fix it

File: README_AR.md [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+[![Discord][discordbadge]][discordurl]
+[![Twitter][twitterurl]][twitterbadge]
+
+<!-- prettier-ignore-start -->
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![جميع المساهمين](https://img.shields.io/badge/all_contributors-11-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+<!-- prettier-ignore-end -->
+
+[عربي](README_ar.md)
```
Comment (jbampton) [inline README_AR.md:31]:
```suggestion
[عربي](README_AR.md)
```

**Comment by BaseMax** [comment]
okay john, you can merge it if it's ready.

File: README_AR.md [hunk 2]
```diff
@@ -0,0 +1,428 @@
+<div align="center">
```
Comment (jbampton) [inline README_AR.md:1]:
```suggestion
<div dir="rtl">

<div align="center">
```

File: README_AR.md [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+    <td align="center"><a href="https://www.upwork.com/freelancers/~013dd1f9db3380689d"><img src="https://avatars.githubusercontent.com/u/81928799?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Mujahid Al-Majali</b></sub></a><br /><a href="https://github.com/One-Language/One/commits?author=Majalian" title="Documentation">📖</a> <a href="#projectManagement-Majalian" title="Project Management">📆</a></td>
+  </tr>
+</table>
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
+
+يتبع هذا المشروع [جميع-المساهمين](https://github.com/all-contributors/all-contributors) مساهمات من أي نوع مرحب بها! 🩱 🕐 1️⃣ 🔂
```
Comment (jbampton) [inline README_AR.md:428]:
```suggestion
يتبع هذا المشروع [جميع-المساهمين](https://github.com/all-contributors/all-contributors) مساهمات من أي نوع مرحب بها! 🩱 🕐 1️⃣ 🔂
</div>
```

**Comment by jbampton** [comment]
I need to fix some `rtl` parts

**Comment by BaseMax** [comment]
Try: <div dir="rtl"></div>

**Review by BaseMax — APPROVED** [review]
Thank you so much.

**Pull request merged**